script_runner/tests/test_ctx_wrapper.py

```diff
@@ -235,7 +258,7 @@ def test_download_resource(self):
     def test_download_resource_with_destination(self):
         fd, temp_path = tempfile.mkstemp(suffix='-resource')
         os.close(fd)
-        script = ('path = ctx.download_resource("resource", "{0}")\n'
+        script = ('path = ctx.download_resource("resource", {0!r})\n'
```

Comment:
the `!r` (repr) escapes the path correctly; on windows the path might contain a `\\1`, so just putting that in a python script would make it puth a literal \x01 byte in the path, which breaks on windows

some encoding is required: repr() is the simplest, but we could also use eg. json, or `.encode('string-escape')` (and then decode inside the script)